client/src/Components/local/atoms/core-mapping/index.tsx

```diff
@@ -202,6 +203,24 @@ const CoreMapping: React.FC<PropTypes> = ({ messages, channel, playerForce, play
           }
         }
       }
+      case 'Text': {
+        const loc = (e as any).layer._latlng as L.LatLng
+        return {
+          type: 'Feature',
+          properties: {
```

Comment:
@hello @lebaphi the `text` element should be of type RENDERER_CORE, rather than RENDERER_MILSYM - since MILSYM is just for entities from the MilSymbol library, please.